src/runner.py

```diff
@@ -85,6 +87,83 @@
 
 app = Flask(__name__)  # pylint: disable=invalid-name
 
+onboarding_authentication_token = ''
+onboarding_authentication_expiry_time = 0
+do_sentry_exception = True
+
+
+def onboarding_authentication_daemon():
```

Comment:
Can you please add these to the `LEDControllerThread` for consistency with other onboarding light calls?